mapApp/models/alert_area.py

```diff
@@ -26,11 +28,11 @@ class Meta:
 class AlertArea(AbstractPolygon):
     """ Class that defines a polygon area of interest where the associated user receives push notifications, and alerts """
     email = models.EmailField(verbose_name="Current email")
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, verbose_name=_("user"))
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, verbose_name=_("user"))
```

Comment:
This newly introduced on_delete method is used to tell Django what to do with model instances that depend on the model instance you delete. (e.g. a ForeignKey relationship). The on_delete=models.CASCADE tells Django to cascade the deleting effect i.e. continue deleting the dependent models as well.